src/test/java/org/mule/extension/sftp/AbstractSftpConnectorTestCase.java

```diff
@@ -18,7 +18,6 @@
         "org.apache.sshd:sshd-common",
         "org.apache.sshd:sshd-scp",
         "org.apache.sshd:sshd-core",
-        "org.bouncycastle:bcprov-jdk15on",
         "net.i2p.crypto:eddsa"
```

Comment:
Considering the comments made in the pom on the issue of whether or not to use EdDSA, I think we should review the use of shared libraries to clearly understand which ones will be present and which ones will not.